README.md

```diff
@@ -1,5 +1,6 @@
 ## Power Up Branch: I2 [![Build Status](https://travis-ci.org/MontclairRobotics/PowerUp.svg?branch=i2)](https://travis-ci.org/MontclairRobotics/PowerUp)
 
+This is the second implementation of GradleRIO and the test of the IntelliJ platorm on Power Up Code for 2018
```

Comment:
This is no longer true

Reply:
@rafibaum he left i2 stuff in there for some reason